### ISSUE TITLE
TestTracingStartSpanCallback tests fail with AssertionError: The setup method 'before_request' can no longer be called on the application.

### ISSUE THREAD

**Issue by mcepl** [opentracing-contrib/python-flask#57]
When packaging 1.1.0 version of the package with Flask 2.3.3 on openSUSE/Tumbleweed, I get the following errors:

```
[   16s] =================================== FAILURES ===================================
[   16s] ___________________ TestTracingStartSpanCallback.test_error ____________________
[   16s] tests/test_flask_tracing.py:218: in test_error
[   16s]     tracing = FlaskTracing(MockTracer(), True, app,
[   16s] flask_opentracing/tracing.py:38: in __init__
[   16s]     def start_trace():
[   16s] /usr/lib/python3.9/site-packages/flask/scaffold.py:49: in wrapper_func
[   16s]     self._check_setup_finished(f_name)
[   16s] /usr/lib/python3.9/site-packages/flask/app.py:511: in _check_setup_finished
[   16s]     raise AssertionError(
[   16s] E   AssertionError: The setup method 'before_request' can no longer be called on the application. It has already handled its first request, any changes will not be applied consistently.
[   16s] E   Make sure all imports, decorators, functions, etc. needed to set up the application are done before running it.
[   16s] ___________________ TestTracingStartSpanCallback.test_simple ___________________
[   16s] tests/test_flask_tracing.py:204: in test_simple
[   16s]     tracing = FlaskTracing(MockTracer(), True, app,
[   16s] flask_opentracing/tracing.py:38: in __init__
[   16s]     def start_trace():
[   16s] /usr/lib/python3.9/site-packages/flask/scaffold.py:49: in wrapper_func
[   16s]     self._check_setup_finished(f_name)
[   16s] /usr/lib/python3.9/site-packages/flask/app.py:511: in _check_setup_finished
[   16s]     raise AssertionError(
[   16s] E   AssertionError: The setup method 'before_request' can no longer be called on the application. It has already handled its first request, any changes will not be applied consistently.
[   16s] E   Make sure all imports, decorators, functions, etc. needed to set up the application are done before running it.
```

[Complete build log](https://github.com/opentracing-contrib/python-flask/files/13607487/_log.txt) with all packages used and steps taken to reproduce.

**Comment by ptmcg** [comment]
I addressed this in PR #60

**Comment by ptmcg** [comment]
Fixed in 2.0.0 release
